action.yml

```diff
@@ -27,17 +27,15 @@ inputs:
   registry_password:
     description: "The registry user password"
     required: false
+  force_create:
+    description: |
+      "Create service forcefully, replaces existing service if any.
```

Comment:
'forcefuly' doesn't really mean anything here.

let's just say 

> Pass `--force` to `kn create` to replace the existing service, instead of failing if the service already exists.